data/en/mbs_0161.json

```diff
@@ -0,0 +1,380 @@
+                                    }
+                                ],
+                                "id": "reporting-period-answer",
+                                "type": "Radio",
+                                "q_code": "d12",
+                                "mandatory": true
+                            }],
+                            "type": "General",
+                            "id": "reporting-period-question",
+                            "title": "Are you able to report for the period from {{exercise.start_date|format_date}} to {{exercise.end_date|format_date}}?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "id": "reporting-period-answer",
+                                        "value": "Yes",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "turnover-block"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "reporting-period-block-2"
+                                }
+                            }
+                        ],
+                        "title": "Reporting period"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
+                            "answers": [{
+                                    "type": "Date",
+                                    "id": "period-from",
+                                    "label": "From",
+                                    "mandatory": true,
+                                    "q_code": "11"
+                                },
+                                {
+                                    "type": "Date",
+                                    "id": "period-to",
+                                    "label": "To",
+                                    "mandatory": true,
+                                    "q_code": "12"
+                                }
+                            ]
+                        }]
+                    },
+                    {
+                        "id": "turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include:",
```

Comment:
I think this should be Turnover for title not 'Total Turnover' as it includes exports. Double check with Donna she said something bout this but slipped my mind.

Reply:
It's inconsistent in slides but total has the majority. Total included as doesn't include a separate Export question.